src/main/java/com/JSONpostroute/validations/ValidationFunctions.java

```diff
@@ -0,0 +1,10 @@
+package com.JSONpostroute.validations;
+
+public class ValidationFunctions {
+    public static boolean reqHasOrdersValidation(String testString){
```

Comment:
I think in C# you can do this arrow function.

public static boolean reqHasOrdersValidation(String testString) => testString.contains("orders") ? true : false